### PR TITLE
emit a warning when connecting with the client tools to a DBSERVER

### PR DESCRIPTION
**Pull request by jsteemann**

### CODE REVIEW

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3431/

File: arangosh/Dump/DumpFeature.cpp [hunk 1]
```diff
@@ -1041,13 +1041,19 @@ void DumpFeature::start() {
...
+  _options.clusterMode = (role == "COORDINATOR");
   if (result.fail()) {
-    LOG_TOPIC(FATAL, arangodb::Logger::RESTORE)
+    LOG_TOPIC(FATAL, arangodb::Logger::DUMP)
         << "Error: could not detect ArangoDB instance type: " << result.errorMessage();
     FATAL_ERROR_EXIT();
   }
 
+  if (role == "DBSERVER" || role == "PRIMARY") {
+    LOG_TOPIC(WARN, arangodb::Logger::DUMP) << "operations in the cluster should be carried via a coordinator, but we are connected to a database server node. this is an unsupported operation!";
```
Comment (Simran-B) [inline arangosh/Dump/DumpFeature.cpp]: Is it really `carried` or should it be `carried out`? Maybe rephrase a little? CC @sleto-it 

```suggestion
    LOG_TOPIC(WARN, arangodb::Logger::DUMP) << "You connected to a DBServer node, but operations in a cluster should be carried out via a Coordinator. This is an unsupported operation!";
```

**Comment by jsteemann** [comment]
@Simran-B : are you happy to suggest the same changes for arangorestore and I will click on "commit suggestion"?

**Comment by jsteemann** [comment]
@Simran-B : adjusted the messages myself.
